client/ayon_deadline/plugins/publish/global/validate_deadline_jobinfo.py

```diff
@@ -36,3 +38,30 @@ def process(self, instance):
         if priority < 0 or priority > 100:
             raise PublishValidationError(
                 f"Priority:'{priority}' must be between 0-100")
+
+        custom_frames = instance.data["deadline"]["job_info"].Frames
+        if not custom_frames:
+            return
+
+        frame_start = (
+            instance.data.get("frameStart")
+            or instance.context.data.get("frameStart")
```

Comment:
Is it necessary to look at context?

Reply:
I noticed that some DCC dont have `instance.data["frameStart"]` (Max, Houdini).